go/libkb/env.go

```diff
@@ -1774,3 +1793,24 @@ func (e *Env) AllowPTrace() bool {
 		func() (bool, bool) { return e.getEnvBool("KEYBASE_ALLOW_PTRACE") },
 	)
 }
+
+func (e *Env) GetLogFileConfig(filename string) *logger.LogFileConfig {
+	megabyte := int64(1024 * 1024)
+	var maxKeepFiles int
+	var maxSize int64
+
+	if e.GetAppType() == MobileAppType && !e.GetFeatureFlags().Admin() {
+		maxKeepFiles = 1
+		maxSize = 32 * megabyte
```

Comment:
too small?

Reply:
I think this is fine since we only send 10mb in log sends right now. Can you add a comment to both places in case we bump the log send or drop this size? https://github.com/keybase/client/blob/master/go/bind/keybase.go#L278